### DISCUSSION TITLE
Nextcloud sync

### DISCUSSION THREAD

**Discussion by doreks** [Eugeny/tabby#5997] [Q&A] [answered]
Hi, firstly my congratulations for the amazing project app.

I would like to know if there is any way to sync or store sessions towards nextcloud or webdav.

Thanks!!

**Comment by Eugeny** [comment, marked as answer]
The `cloud-setttings-sync` plugin lets you sync to a WebDAV share

**Comment by lawweiliang** [comment]
@doreks 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
